_includes/tutorials/transforming/ksql/markup/dev/transient-query.adoc

```diff
@@ -1,4 +1,4 @@
-Let's break apart the `title` field and extract the year that the movie was published into its own column. Issue the following transient query. This will block and continue to return results until it's limit is reached or you tell it to stop.
+Let's break apart the `title` field and extract the year that the movie was published into its own column. Issue the following transient push query. This will block and continue to return results until it's limit is reached or you tell it to stop.
```

Comment:
```suggestion
Let's break apart the `title` field and extract the year that the movie was published into its own column. Issue the following transient push query. This will block and continue to return results until its limit is reached or you tell it to stop.
```